src/Listener.js

```diff
@@ -0,0 +1,63 @@
+const desktopIdle = require('desktop-idle');
+
+// We want the status of the user to be accurate.
+// To ensure this we need to frequently check the users status
+const DEFAULT_POLLING_TIMEOUT = 1000;
+
+class Listener {
+  constructor(id, timeToAway, callback) {
+    this.id = id;
+    this.timeToAway = timeToAway;
+    this.timeoutId = null;
+    this.callback = callback;
+    this.isAway = false;
+    this.shouldListen = true;
+  }
+
+  checkIsAway() {
+    let idleSeconds;
+
+    try {
+      idleSeconds = desktopIdle.getIdleTime();
+    } catch (error) {
+      this.callback(null, error);
```

Comment:
We should probably switch the order of the args in the callback. The general convention in `node.js` is to have [error first callbacks](https://nodejs.org/api/errors.html#errors_error_first_callbacks)

Reply:
That should fine, this is a major version bump.